map_gui/src/render/car.rs

```diff
@@ -216,10 +220,12 @@ impl Renderable for DrawCar {
         g.redraw(&self.draw_default);
     }
 
-    fn get_outline(&self, _: &Map) -> Polygon {
-        self.body
-            .to_thick_boundary(CAR_WIDTH, OUTLINE_THICKNESS)
-            .unwrap_or_else(|| self.body_polygon.clone())
+    fn get_outline(&self, _: &Map) -> Tessellation {
```

Comment:
Very slight change here.
![Screenshot from 2022-08-30 15-49-44](https://user-images.githubusercontent.com/1664407/187468853-d7668970-3fd3-4509-9d92-b682c44e384f.png)
![Screenshot from 2022-08-30 15-49-28](https://user-images.githubusercontent.com/1664407/187468863-f6fc8a8b-f55c-4ec2-aa14-2cedc38836bf.png)
Most of the time, the body polygon narrows in the front. The outline has always just been the simpler rectanglish shape. Before, the hitbox was based on the more detailed narrowed shape. But now the hitbox is also just the simple rectangle. Barely noticeable.